src/generation/generate.rs

```diff
@@ -7096,11 +7102,8 @@ where
             single_line_space_at_start: space_around,
             single_line_space_at_end: space_around,
             custom_single_line_separator: None,
-            multi_line_options: if prefer_single_item_hanging {
-              MultiLineOptions::maintain_line_breaks()
-            } else {
-              MultiLineOptions::surround_newlines_indented()
-            },
+            // If we end up in multiline mode, print `call(\n...\n)`
+            multi_line_options: MultiLineOptions::surround_newlines_indented(),
```

Comment:
This was the original behaviour before [my previous incorrect PR.](https://github.com/dprint/dprint-plugin-typescript/pull/472/files#diff-006be3bb05abfb2221b2909f0e968999c52012e8c7eb6a6ed245df44238d18b6L7059)